### ISSUE TITLE
As a user I need to be able to navigate to the christmas tree portions of the permit web site from the home page.

### ISSUE THREAD

**Issue by sadlerw** [nciinc/fs-permit-platform#263]
##Notes
- do we want this? 
- pilot forests are different for each permit/xmas tree permits... do we want to separate into two home pages to reduce confusion?

**Comment by sadlerw** [comment]
@ASprinkle @lauraGgit @hannahkane let's make sure we discuss this at the sprint planning meeting.  I think that depending on the decision here (or maybe not) we should also create a story for a page that sits before the "find a forest for xmas tree permits" page in the user flow that could be the xmas tree main page -- similar to the first page of a brochure for example.  Friendly, warm, inviting, etc.  A pic something like the front of the flathead brochure for example, etc.

**Comment by ASprinkle** [comment]
Agreed, a discussion is a good idea. A welcome page would be nice.  If we do this, the welcome page should offer a choice.  A user (new to the process) could choose to select their forest and read all of the information on Christmas trees, or an experienced Christmas tree gatherer could simply select the buy permit option for a forest and move through the process quickly.

**Comment by sadlerw** [comment]
I liked your idea about adding the buy a permit button under the scrolling menu on the tree guidelines page which also helps to address the issue of the sophisticated tree harvester vs the neophyte.  We should get that in somewhere too.

**Comment by ASprinkle** [comment]
Thank you - I think the team has already done some research, and I think @bryanmcfadden has already done usability testing on the "Buy Permit" button(?) - I would like to know what their perspective is on this before moving forward.

**Comment by MelissaBraxton** [comment]
Clarifying Q - Now that we've started working #256 and #257, I'm wondering whether we still need this. @ASprinkle if you think #256 and #257 are a good first take on what this story is getting at, can we archive #263?

**Comment by ASprinkle** [comment]
@MelissaBraxton Agreed, we should close/archive this story, as the work on 256 and 257 appear to be meeting the goal of this story.  This story seems redundant and not as specific as 256 and 257.  Good call.
